src/viam/media/video.py

```diff
@@ -1,15 +1,25 @@
+from .viam_rgba_plugin import RGBA_FORMAT_LABEL
 from enum import Enum
 from io import BytesIO
-from typing import NamedTuple, Union
-
+from typing import NamedTuple, Union, Tuple
+from typing_extensions import Self
 from PIL.Image import Image
 
+LAZY_SUFFIX = "+lazy"
+
+# Formats that are supported by PIL
+LIBRARY_SUPPORTED_FORMATS = ["JPEG", "PNG", RGBA_FORMAT_LABEL]
+
 
 class RawImage(NamedTuple):
     """A raw bytes representation of an image.
 
-    When requesting `CameraMimeType.RAW` or if the camera returns a mime type that is not supported,
-    the returned type will be this RawImage type.
+    A RawImage should be returned instead of a PIL Image instance under one of
+    the following conditions
+
+    1) The requested mime type has the LAZY_SUFFIX string appended to it
+    2) The requested mime type is not supported for decoding/encoding by Viam's
+    Python SDK
```

Comment:
From what I understand, LAZY is only for while in transit. If your python SDK is the final destination, it should probably decoded, even if its LAZY. @njooma  Will there be times when the python SDK will be an intermediate hop and be used as a server for a camera as well? i.e. python will be sending images it has received from another server.

Reply:
Is there is a chance that python could be a server, then what's you're doing is right Gautham, and anything with the lazy suffix should just return the raw bytes

Reply:
I don't see why a user couldn't use the python-sdk as a camera server -- it definitely has that capability to provide images as well as consume them